### DISCUSSION TITLE
error when termiral runs gnuradio-companion

### DISCUSSION THREAD

**Discussion by kjasflrefvkjr** [community/community#118878] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![Screenshot from 2024-04-13 11-09-38](https://github.com/community/community/assets/166891848/c3219c51-b6d7-4f85-951f-b5b9eb8ac013)
When I run gnuradio-companion on terminal, I get a library missing error. I fixed it according to the instructions on https://wiki.gnuradio.org/index.php/ModuleNotFoundError, I run the command "gnuradio-config-info --prefix" and the terminal displays "gnuradio-config-info: error while loading shared libraries: libgnuradio-runtime.so.3.10.7: cannot open shared object file: No such file or directory" can anyone help me fix it? Thanks

**Comment by ettaboyle** [comment]
Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
